docs/metrics.md

```diff
@@ -0,0 +1,142 @@
+
+# Tab Split Metrics
+
+Metrics collections and analysis plan for Tab Split as part of the [Firefox Test Pilot program](https://testpilot.firefox.com).
+
+## Analysis
+
+Data collected in this experiment will be used to answer the following high-level questions:
+
+* How often do users load a non-homepage in the sidebar?
+* Are people who use Tab Split heavy tab users?
+
+
+
+## Collection
+Data will be collected with Google Analytics and follow [Test Pilot standards](https://github.com/mozilla/testpilot/blob/master/docs/experiments/ga.md) for reporting.
+
+## Custom Metrics
+*none*
+
+### Custom Dimensions
+
+* `cd1` - The width of the sidebar in pixels.  An integer
+* `cd2` - The number of open tabs in the current window.  An integer
+* `cd3` - The site type requested.  One of `desktop` or `mobile`
+
+### Events
+
+#### `Interface`
+
+##### When the user chooses to load a link in the sidebar from the context menu
+```
+ec: interface,
+ea: load-url,
+el: context-menu,
+cd1,
+cd2,
+cd3
+```
+
+##### When the user chooses to load a link in the sidebar by choosing an existing tab on the home page
+```
+ec: interface,
+ea: load-url,
+el: existing-tab,
+cd1,
+cd2,
+cd3
+```
+
+##### When the user chooses to load a link in the sidebar by choosing a previously loaded tab on the home page
+```
+ec: interface,
+ea: load-url,
+el: previous-tab,
+cd1,
+cd2,
+cd3
+```
+
+##### When the user activates the Tab Split back button
+```
+ec: interface,
+ea: button-click,
+el: button-back,
+cd1
+```
+
+##### When the user activates the Tab Split refresh button
+```
+ec: interface,
+ea: button-click,
+el: button-refresh,
+cd1,
+cd3
+```
+
+##### When the user activates Tab Split from the Sidebar dropdown menu
+```
+ec: interface,
+ea: button-click,
+el: sidebar-menu,
+cd1,
+cd2
+```
+
+##### When the user activates Tab Split from a keyboard shortcut
+```
+ec: interface,
+ea: button-click,
+el: keyboard-shortcut,
+cd1,
+cd2
+```
+
+##### When the user toggles the desktop or mobile request checkbox
+```
+ec: interface,
+ea: button-click,
+el: request-desktop  **or**  request-mobile,
+cd1
+```
+
+##### When the user clicks the feedback button
+```
+ec: interface,
+ea: button-click,
+el: feedback,
+cd1,
+cd2,
+cd3
+```
+
+#### `Content`
+
+##### When the home page loads for whatever reason
+```
+ec: content,
+ea: load-url,
+el: home-page,
+cd1,
+cd2
+```
+
+##### When a child page loads
+```
+ec: content,
+ea: load-url,
+el: child-page,
+cd1,
+cd2,
+cd3
+```
+
+##### When a link is clicked in a child page and the main window is asked to load it
```

Comment:
I don't think we'll naturally see this event (though maybe through judicious use of `tabs.onCreated` or maybe `webNavigation.onCreatedNavigationTarget` we may be able to see tabs created by tabId -1)

Reply:
Maybe I'm not explaining this well.  I mean someone loads twitter in the left sidebar and then clicks a link and it opens in the right side main window.  Couldn't we just fire an event when they click the link?